Source/events/EventLogSequenceNumberMustBeNaturalNumber.ts

```diff
@@ -1,5 +1,6 @@
 // Copyright (c) Dolittle. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
 import { Exception } from '@dolittle/rudiments';
 
 /**
```

Comment:
I think natural number as a concept isn't obvious for most people, a better name is probably `EventLogSequenceNumberMustBeAPositiveInteger`

Reply:
done